### DISCUSSION TITLE
How to build DuckDB to get the duckdb windows cli

### DISCUSSION THREAD

**Discussion by ravi27may** [duckdb/duckdb#2705] [Q&A]
Hi,

Am trying to read s3 parquet file using duckdb windows cli, but it seems it doesn't working. After reading few discussion understand that we need to build duckdb with httpfs extension enabled. I didn't find on how to build this for windows and my c knowledge is very little. Could someone please help

Regards,
Ravi

**Comment by Mytherin** [comment]
We are working on a solution for this with installable extensions (#2569). The HTTPFS extension is not supported on Windows yet in general, however.

**Comment by ravi27may** [comment]
Thanks Mytherin
